working-group/notes/2024/2024-07.md

```diff
@@ -1,4 +1,4 @@
-# GraphQL-over-HTTP WG - 25th January 2024
+# GraphQL-over-HTTP WG - 25th July 2024
 
 **Watch the replay**:
 https://www.youtube.com/watch?v=nHSixplvCc0&list=PLP1igyLx8foEz9127xc0SsabIrbTMt9g5
```

Comment:
Note: Link currently points to an older video, although it is one within the proper playlist.